### PR TITLE
Aesthetic improvement to welcome page

### PR DESCRIPTION
**Pull request by agzuniverse**
Side by side of before/after:
welcome_sidebyside

Thoughts?

### CODE REVIEW

**Comment by borisyankov** [comment]
I have no preference for either of them. 
I think the start screen can be improved a lot but that would likely involve a graphics designer doing some cool graphics.

**Comment by agzuniverse** [comment]
Seems like that is the better choice. I dropped the commit that added the `hideTitle` prop to screen and used a `View` with `styles.screen` instead.

**Comment by agzuniverse** [comment]
@borisyankov further review?

File: src/start/WelcomeScreen.js [hunk 1]
```diff
@@ -2,36 +2,63 @@
...
 
 import React, { PureComponent } from 'react';
+import { StyleSheet, View } from 'react-native';
 
-import type { Dispatch } from '../types';
-import { Screen, ViewPlaceholder, ZulipButton } from '../common';
+import type { Context, Dispatch } from '../types';
+import { ViewPlaceholder, ZulipButton, Label, Logo } from '../common';
 import { navigateToAddNewAccount, navigateToWelcomeHelp } from '../actions';
+import StylesProvider from '../boot/StylesProvider';
```
Comment (borisyankov) [inline src/start/WelcomeScreen.js]: You don't need this import.

**Comment by agzuniverse** [comment]
Updated, removing the unused import.

**Comment by gnprice** [comment]
Thanks @agzuniverse for this PR!

@borisyankov and I just looked at it together, and as he said above we're not sure this is better than the existing screen. It would also definitely need some testing on small screens to make sure it still worked reasonably well.

We might at some point revisit this screen, but as Boris said above:

> that would likely involve a graphics designer doing some cool graphics.

**Comment by agzuniverse** [comment]
Okay, that's cool!

**Pull request closed without merging**